core/Policy/PolicyManager.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Piwik\Policy;
+
+use Exception;
+use Piwik\Plugin\Manager;
+use Piwik\Settings\Interfaces\PolicyComparisonInterface;
+use Piwik\Settings\Interfaces\SettingValueInterface;
+
+class PolicyManager
+{
+    /**
+     * @return array<class-string<CompliancePolicy>>
+     */
+    public static function getAllPolicies(): array
+    {
+        return [
+            CnilPolicy::class,
+            HipaaPolicy::class
+        ];
+    }
+
+    /**
+     * @return array<array<string, string>>
+     */
+    public static function getAllPoliciesDecorated(): array
```

Comment:
Perhaps consider renaming to as we are not adding any other details than we already have in each policy?

```suggestion
    public static function getAllPoliciesDetails(): array
```

Reply:
updated the function name.